src/cdk/collections/selection.ts

```diff
@@ -171,3 +180,8 @@ export class SelectionModel<T> {
 export class SelectionChange<T> {
   constructor(public added?: T[], public removed?: T[]) { }
 }
+
+/** Throws an error if multiple values are passed into a selection model with a single value. */
+export function throwMultipleValuesInSingleSelectionError() {
+  throw Error('Cannot pass multiple values into SelectionModel with single-value mode.');
```

Comment:
Since it is only being used in one place, consider moving this inline into `_warnMultipleValuesForSingleSelection`?

Reply:
I think we have a convention that we create functions for these errors and re-export the Error object. 

e.g in https://github.com/angular/material2/blob/master/src/lib/select/select-errors.ts#L28

<sub>Updating to follow that convention</sub>

Reply:
Those errors are in a function so we can use them in tests. If it's a one-off, it's fine to have it inline.